librz/core/disasm.c

```diff
@@ -885,9 +885,29 @@ static void __replaceImports(RzDisasmState *ds) {
 	}
 }
 
+static void ds_opstr_try_colorize(RzDisasmState *ds, bool print_color) {
```

Comment:
If you decide to port Tricore to tokens, please keep this function extraction change - it's still useful.